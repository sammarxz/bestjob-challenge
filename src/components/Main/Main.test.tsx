import React from 'react'
import { render } from '@testing-library/react'

import { Main } from './'

describe('<Main />', () => {
  it('should render the <h1> as children of component', () => {
    const { container, getByText } = render(
      <Main>
        <h1>BestJob</h1>
      </Main>
    )

    expect(getByText('BestJob')).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot(`
      <h1>BestJob</h1>
    `)
  })
})
