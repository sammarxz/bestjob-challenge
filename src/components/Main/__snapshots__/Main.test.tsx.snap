// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the <h1> as children of component: 
      <h1>BestJob</h1>
     1`] = `
<div
  class="undefined flex flex-col flex-wrap items-center justify-center"
>
  <main
    class="container max-w-4xl mx-auto px-4 md:px-8"
  >
    <h1>
      BestJob
    </h1>
  </main>
</div>
`;
